src/game-of-life/time-profiling.md

```diff
@@ -29,7 +29,8 @@ measurements from it without distorting the performance of the rest of the
 system.
 
 For example, we can create a simple frames per second (FPS) counter that we
-update on each iteration of our `renderLoop`:
+update on each iteration of our `renderLoop`.
+Create `fps.js` then add :
```

Comment:
In the repo for the tutorial's code, we don't create a separate file/module for `fps`, so we should be consistent with that here so that folks don't get confused when reading the final source: https://github.com/rustwasm/wasm_game_of_life/blob/acb9a67ff0df50815a85f643438774d5f7128a99/index.js#L27-L63